src/main/drivers/vtx_smartaudio.c

```diff
@@ -0,0 +1,706 @@
+#include <stdbool.h>
+#include <stdint.h>
+#include <ctype.h>
+
+#include "platform.h"
+
+#ifdef VTX_SMARTAUDIO
+
+#include "drivers/system.h"
+#include "drivers/serial.h"
+#include "drivers/vtx_smartaudio.h"
+#include "io/serial.h"
+#include "fc/runtime_config.h"
+#include "config/config_master.h"
```

Comment:
I think I can live without fc/runtime_config.h and config/config_master.h.

io/serial.h is included for
1. To find out a port assigned to the smartaudio.
2. To open a uart port for printf() debugging.

2 will eventually go away, but where should I put 1? main.c?